### PR TITLE
Added Twitter Link 

### LINKED ISSUE #480: Add Twitter to Connect
### What I expected to happen
I expected to see a Twitter icon in the "Connect" section of the website footer, allowing users to connect with the organization via Twitter.

### Our reason
Adding a Twitter icon/link would enhance the visibility of the organization's social media presence and provide an additional platform for users to stay updated and engage with the organization's content.

### What I plan to do 
1. Add a Twitter icon alongside the existing icons in the "Connect" section.
2. Link the icon to the organization's official Twitter profile.
3. Ensure the icon design matches the style of the existing icons for consistency.

## current Connect Section 
Screenshot from 2024-08-06 19-56-40

### PR DESCRIPTION
**Pull request by Indrajeety993648**
## Description

This pull request adds the Twitter logo to the relevant components using React Icons. The logo is used for social media links and enhances the visual appeal of the application.

## Changes

- Imported the Twitter logo from `react-icons/bs`.
- Updated the relevant component(s) to include the Twitter logo.

## Motivation and Context

Adding the Twitter logo provides a recognizable icon for linking to Twitter profiles or posts. This integration is part of the effort to improve social media connectivity in the application.

## How Has This Been Tested?

- Verified that the logo appears correctly in the designated components.
- Ensured the responsiveness and visual alignment of the logo on different screen sizes.

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.

## Related Issue

Fixes # 480
## Screenshots 

Screenshot from 2024-08-07 17-32-14

### CODE REVIEW

**Comment by Indrajeety993648** [comment]
@rithviknishad  please review  the pull request .

File: components/Footer.tsx [hunk 1]
```diff
@@ -114,6 +116,11 @@ export default function Footer() {
                   icon={FaEnvelope}
                   label="Email"
                 />
+                <SocialLink
+                  href={`mailto:${env.NEXT_PUBLIC_TWITTER_URL}`}
```
Comment (rithviknishad) [inline components/Footer.tsx:120]: Why is the url prefixed with `mailto:`?

File: components/Footer.tsx
Comment (rithviknishad) [inline components/Footer.tsx:1]: As mentioned in the issue already, make all of the social links optional, and skip rendering it if it is not defined.

File: components/Footer.tsx (hunk 1 above)
Comment (rithviknishad) [inline components/Footer.tsx:120]: This needs to be added to the env schema too

File: pnpm-lock.yaml
Comment (rithviknishad) [inline pnpm-lock.yaml:1]: Could you discard the changes to the lockfile?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml
Comment (Indrajeety993648) [reply pnpm-lock.yaml:1]: @rithviknishad sure

**Comment by rithviknishad** [comment]
Closing as duplicated of #482

**Pull request closed without merging**